kornia/testing/__init__.py

```diff
@@ -303,7 +303,7 @@ def KORNIA_CHECK_SHAPE(x, shape: List[str]) -> None:
     """
     # Desired shape here is list and not tuple, because torch.jit
     # does not like variable-length tuples
-    KORNIA_CHECK_IS_TENSOR(x)
+    # KORNIA_CHECK_IS_TENSOR(x)
```

Comment:
@johnnv1 we have an issue here because the Tensorwrapper subclasses are not detected as Tensor instance. Tried to explicit import Vector3 here but issues with circular dependencies

Reply:
there are other ways like to check .shape arribute but then our firend torchscript complains

Reply:
i think testing `(Tensor, TensorWrapper)` at `KORNIA_CHECK_IS_TENSOR`, should work

Reply:
torchscript complain about it

lets hack this around and we get rid of torchscript tests after the release